libipl/p10/ipl0.C

```diff
@@ -344,6 +344,58 @@ static void process_guard_records()
 	}
 }
 
+//@Brief Function will check if the FCO state is set for master proc
+// children or not. If FCO override bit is set for children, need to mark
+// present and functional state to set in devicetree during the boot.
+static bool update_master_proc_fco_state(void)
```

Comment:
Function name is misleading. repalce name to applyFCOOverride().
Also add not saying scope is limited to primary processor assciated hw usints consumed By sbe.

Reply:
Done.